graphql-kotlin-spring-server/src/main/kotlin/com/expediagroup/graphql/spring/GraphQLConfigurationProperties.kt

```diff
@@ -45,4 +45,6 @@ class FederationConfigurationProperties {
 class SubscriptionConfigurationProperties {
     /** GraphQL subscriptions endpoint, defaults to 'subscriptions' */
     var endpoint: String = "subscriptions"
+    /** Keep the websocket alive and send a message to the client every interval in ms. Default to not sending messages */
+    var keepAliveInterval: Long? = null
```

Comment:
unsure if it is needed but can you also add it in to https://github.com/ExpediaGroup/graphql-kotlin/blob/master/graphql-kotlin-spring-server/src/main/resources/META-INF/spring-configuration-metadata.json